src_py/_sdl3_mixer.py

```
@@ -3,7 +3,7 @@
 from pygame import _audio as audio, _sdl3_mixer_c  # pylint: disable=no-name-in-module
 
 init = _sdl3_mixer_c.init
-# quit = _sdl3_mixer_c.quit
+quit = _sdl3_mixer_c.quit
```

Ruff (0.16.2)

[error] Variable `quit` is shadowing a Python builtin

(A001)